common/smoke-test/Initialize-SmokeTests.ps1

```diff
@@ -113,12 +121,10 @@ function New-DeployManifest {
 function Update-SamplesForService {
   Param([Parameter(Mandatory = $true)] $entry)
 
-  Write-Verbose "Preparing samples for $($entry.Name)"
-  dev-tool samples prep --directory $entry.PackageDirectory --use-packages
-
-  # Resolve full path for samples location. This has to be set after sample
-  # prep because the directory will not resolve until the folder exists.
-  $entry.SamplesDirectory = Join-Path -Path $entry.PackageDirectory -ChildPath 'dist-samples/javascript' -Resolve
+  $sampleFiles = Get-ChildItem "$($entry.SamplesDirectory)/*.js"
+  foreach ($sampleFile in $sampleFiles) {
+    (Get-Content -Raw $sampleFile) -replace "(?s)main\(\)\.catch.*", "module.exports = { main };`n" | Set-Content -Path $sampleFile
```

Comment:
While it might be fun to make this a nice one-liner can we break it up so it is a little easier to understand?